source/tmpl/settings.gohtml

```diff
@@ -37,6 +37,20 @@
                                 <input type="text" name="description" class="form-control" value="{{ .Description }}" id="description" placeholder="Great Uptime">
                             </div>
 
+                            <div class="form-group">
+                                <div class="col-4 col-sm-4 mt-sm-1 mt-0">
+                                    <label for="update_notify" class="d-inline d-sm-none">Send Updates only</label>
```

Comment:
Wow, it's so not intuitive, could you please improve the wording? Thanks. I'll submit separated issue for that

Reply:
I'm currently working on another PR which also _"fixes"_ this "_problem"_.
It will also include a _tooltip_ so a User can understand better.

So please be patiently.

Reply:
Sure, no worries :) See #301 btw

Reply:
The Discussion moved to your Issue (#301) so I will resolve this conversation even if it's not _really_ resolved.